src/openzaak/components/documenten/api/serializers.py

```diff
@@ -1112,3 +1114,47 @@ class Meta:
             "url": {"lookup_field": "uuid", "read_only": True},
         }
         validators = [VerzendingAddressValidator()]
+
+
+class ReservedDocumentSerializer(serializers.ModelSerializer):
```

Comment:
To make sure this can also be used via the API, we'll need to have a view that uses this serializer. The reserveer zaken endpoint has one as well: https://github.com/open-zaak/open-zaak/blob/main/src/openzaak/components/zaken/api/viewsets.py#L1431 which is also registered in `urls.py` to make sure it's available at a path.

If possible I'd like to see if we can use a function based view (https://www.django-rest-framework.org/api-guide/views/#function-based-views) for this instead of a class based view (like the reserveer zaaknummer endpoint has now), because it's just a single method (POST), but let's first implement it the same as reserveer zaaknummer and check if we can refactor this later